src/components/FileBrowser.js

```diff
@@ -162,7 +178,8 @@ class FileBrowser extends React.Component {
     const { fileContentMap } = this.props;
     const fileContent = _.get(fileContentMap, this.state.selectedFile, '');
 
-    const extension = this.state.selectedFile.match(extensionRegex);
+    const extensionMatch = this.state.selectedFile.match(extensionRegex);
+    const extension = Array.isArray(extensionMatch) ? extensionMatch[0] : null;
```

Comment:
We use lodash in this project, consider write this in lodash for consitency. By the way, is there a eslint rule for this?

Reply:
@jakoblind 
Used lodash version if this function in 1f8570f651daa666bc31f436cb462f8c0e8142d7 . BTW, `extensionMatch ? extensionMatch[0] : null;` should work exactly the same in this case, dunno why I like `isArray` ;)

Reply:
@jakoblind 
Re eslint; just found sth like this: https://github.com/wix/eslint-plugin-lodash/blob/HEAD/docs/rules/prefer-lodash-method.md , but I dunno whether you were thinking about sth like that.

Reply:
Another option is to use [_.head](https://lodash.com/docs/4.17.15#head). It returns `undefined` if it's not an array, or an empty array. I think it's a bit more readable, but you decide.

Yes those eslint rules was what I was thinking about! Will look into that

Reply:
@jakoblind 
Like it. Updated.